### DISCUSSION TITLE
Mobile App Request

### DISCUSSION THREAD

**Discussion by furkanayilmaz** [forem/forem#17534] [Icebox]
The platform it’s self is very good! However, in my opinion people mostly login from the mobile so there should be a mobile application. Well there already is one but it is not good. It is using web view to show the web page. There is no difference then logging in from Safari and downloading the app.

**Comment by ranggakd** [comment]
Now I know why the mobile app behaves that way. I thought it could direct you to the mobile app from the mobile web browser as Twitter did.

When you open the Twitter link in a mobile web browser and have Twitter as a mobile app, you could see the button to open it in your mobile app instead of your web browser

![Twitter from mobile browser](https://user-images.githubusercontent.com/36787606/201385160-c6bbc6ee-4851-4ea9-bfa8-6efb6a2f853e.jpg)

I wonder if Dev could do the same

![Dev from mobile browser](https://user-images.githubusercontent.com/36787606/201385860-6f58c587-e755-4459-8c0f-24ebd43591b2.jpg)
